razorpay-sdk/src/Utility.php

```diff
@@ -9,10 +9,20 @@ class Utility
     public function verifyPaymentSignature($attributes)
     {
         $expectedSignature = $attributes['razorpay_signature'];
-        $orderId = $attributes['razorpay_order_id'];
         $paymentId = $attributes['razorpay_payment_id'];
 
-        $payload = $orderId . '|' . $paymentId;
+        if (isset($attributes['razorpay_order_id']) === true)
+        {
+            $orderId = $attributes['razorpay_order_id'];
+
+            $payload = $orderId . '|' . $paymentId;
+        }
+        else
```

Comment:
You can do `if()elseif()else(throw error)`. Makes for more robust code. Not deal breaking, but I'd recommend. This also makes a lot of sense for when we'll be porting these changes to the razorpay-php sdk (where you don't know what the client can/will send across)